ERCS/erc-7620.md

```diff
@@ -0,0 +1,177 @@
+---
+eip: 7620
```

Comment:
```suggestion
eip: 7620
```
Assigning next sequential EIP/ERC/RIP number.  7620 was assigned to an EIP.  Numbers are assigned by EIP/RIP/ERC editors/associates.  Number sniping is not allowed.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename and reference contracts.